src/app/components/cards/PostsList.jsx

```diff
@@ -211,12 +211,12 @@ class PostsList extends React.Component {
 
 export default connect(
     (state, props) => {
-        const pathname = state.app.get('location').pathname;
-        const current = state.user.get('current')
-        const username = current ? current.get('username') : state.offchain.get('account')
-        const content = state.global.get('content');
-        const ignore_result = state.global.getIn(['follow', 'getFollowingAsync', username, 'ignore_result']);
-        const userPreferences = state.app.get('user_preferences').toJS();
+        const pathname = state.getIn(['app', 'location']).pathname;
```

Comment:
this is an argument in favor of option 1 for event `type`s as described above

Reply:
not sure what this line has to do with actions -- selecting from the store here